manifest_staging/deployment/provider-vault-installer.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: secrets-store-csi-driver-provider-vault
```

Comment:
Now that the repo is renamed, `s/secrets-store-csi-driver-provider-vault/vault-csi-provider/` in this file, except for the docker image name.